File: Year 2/Semester 4/Programming and Projection Enironments/Week 10/ProtobuffProjectJavaORM/Persistance/src/main/java/gRPCProject/persistance/repository/ITicketRepository.java
package gRPCProject.persistance.repository;

import gRPCProject.domain.Entity;
import gRPCProject.domain.Ticket;

import java.io.Serializable;

public interface ITicketRepository <ID extends Serializable, E extends Entity<ID>> extends IRepository<ID, E> {
    void addTicket(Ticket ticket);
}

File: Year 2/Semester 4/Programming and Projection Environments/Week 9/ProtobuffProjectCSharp/persistence/interfaces/ITicketRepository.cs
using ProtobuffProject.domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProtobuffProject.repository.interfaces
{
    public interface ITicketRepository : IRepository<int, Ticket<int>>
    {
        void AddTicket(Ticket<int> ticket);
    }
}
